app/views/applications/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<h1>Name: <%= @application.name %></h1>
+<h3>Address: <%= @application.address %></h3>
+<h3>Description: <%= @application.description %></h3>
+<h3>Status: <%= @application.status %></h3>
+
+<h2>Links To Pets On Application</h2>
+
+<% @application.pets.each do |pet| %>
+  <ul>
+    <li><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></li>
+  </ul>
+<% end %>
+
+<% if @application.has_pet? %>
+  <h3>Reason for adopting this pet:</h3>
+
+  <%= form_with url: "/applications/#{@application.id}", method: :patch, data: { turbo: false }, local: true do |form| %> 
+    <%= form.label :reason %>
+    <%= form.text_field :reason %> 
+      <br></br>
+    <%= form.submit "Submit Application" %>
+  <% end %>
+
+  <p>No pets associated with this application.</p>
+<% end %>
+
+<% if @application.status == "In Progress" %>
+  <h4>Add a Pet to this Application</h4>
+
+  <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |form| %>
+    <%= form.label :search %>
+    <%= form.text_field :search %> 
+      <br></br>
+    <%= form.submit "Search Pets" %>
+  <% end %>
+<% end %>
+
+<% if @application.status == "Pending" %>
+  <p>Application is Pending</p>
+<% end %>
+
+<% if @result %>
+  <% @result.each do |pet| %>
+    <ul>
+      <li><%= pet.name %></li>
+      <li>
+        <%= form_with url: "/applications/#{@application.id}/pets/#{pet.id}", method: :patch, local: true do |form| %>
+          <%= form.submit "Adopt this Pet" %>
+        <% end %>
```

Comment:
Whenever you find yourself creating a form just to hold a button, then it is probably a button that you want and not a form.